app/views/layouts/application.html.slim

```diff
@@ -18,28 +18,29 @@ html lang="#{I18n.locale || I18n.default_locale}" data-default-locale="#{I18n.de
     meta name='current-user' content=current_user&.to_page_context&.to_json
   body.d-flex.flex-column.h-100
     #content
-      nav.navbar.bg-light.navbar-expand-lg.mb-0 role='navigation' data-bs-theme="light"
-        .container
-          = link_to home_index_path, class: "navbar-brand"
-            i.fa-solid.fa-anchor.me-2#logo
-            = application_name
-          button.navbar-toggler data-bs-target='#navbar-collapse' data-bs-toggle='collapse' type='button' aria-expanded='false' aria-label='Toggle navigation' aria-controls='navbar-collapse'
-            span.navbar-toggler-icon
-          #navbar-collapse.collapse.navbar-collapse
-            = render('navigation')
-            ul.nav.navbar-nav.ms-auto
-              = render('session')
+      header.py-3
+        nav.navbar.bg-light.navbar-expand-lg.fixed-top.text-bg-light.fs-6-p-0 role='navigation'
+          .container
+            = link_to home_index_path, class: "navbar-brand"
+              i.fa-solid.fa-anchor.me-2.text-secondary
+              = application_name
+            button.navbar-toggler.rounded-0.m-2 data-bs-target='#navbar-collapse' data-bs-toggle='collapse' type='button' aria-expanded='false' aria-label='Toggle navigation' aria-controls='navbar-collapse'
+              span.navbar-toggler-icon
+            #navbar-collapse.collapse.navbar-collapse
+              = render('navigation')
+              ul.navbar-nav.ms-auto
+                = render('session')
       div data-controller=controller_name
         = render('flash')
         span.d-none = yield(:breadcrumbs)
-      .container
-        .row
-          - if controller_name == "home" && action_name == "index"
-            .col-md-12.p-2
-              = yield
-          - else
-            .col-md-12.p-4 style=("margin-top: 32px")
-              = yield
+      .row
+        - if controller_name == "home" && action_name == "index"
+          .col-md-12.p-2
+            = yield
```

Comment:
I could not think more elegant solution,  rather than to put it this way.

On the `home` page, I have to put `container` div in each sections, in order to be able to work on the whole width styling (such as the whole services-section bg covered with grey color) but this kind of styling only implemented on the home page, and not on the other pages. 

The other workaround is to remove the "if" in the application layout, but put `container` div on every other pages except the home page.